### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Miracle8221**
Simple react page

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { SideBar } from './components/SideBar'
+import Body  from './components/Body'
+import Card from './components/Card'
+import  Footer  from './components/Footer'
+
+
+function App() {
+  return (
+    <>
+    <div className=' w-full h-screen'>
```
Comment (Phildeoner) [inline src/App.jsx]: Work on your indentation, you can install prettier to help you handle the formatting

File: src/components/Body.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import image from "../assets/image1.jpg"
+
+function Body () {
+  return (
+    <div>
+    <div className='font-bold'>
```
Comment (Phildeoner) [inline src/components/Body.jsx]: Review this indentation too

**Review by Phildeoner — APPROVED** [review]
Hi @Miracle8221 👋, Well done you did a great job, just pay more attention to your code formatting and indentations. You can proceed to merge and wrap up this task.
Happy Coding!😊😊

File: src/App.jsx (hunk 1 above)
Comment (Miracle8221) [reply src/App.jsx]: alright thanks i will install the prettier

**Pull request merged**